lib/_http_common.js

```diff
@@ -14,8 +14,8 @@ const debug = require('util').debuglog('http');
 exports.debug = debug;
 
 exports.CRLF = '\r\n';
-exports.chunkExpression = /chunk/i;
-exports.continueExpression = /100-continue/i;
+exports.chunkExpression = /(?:^|\W)chunked(?:$|\W)/i;
+exports.continueExpression = /(?:^|\W)100-continue(?:$|\W)/i;
```

Comment:
It's more explicit and therefore at least a little safer and it's consistent with the other regexps we use in _http_server.js when searching header values that can be comma-separated lists.

I did not measure the performance of this particular change on its own.